site/content/docs/manifest/rd-web-service.ja.md

```diff
@@ -138,8 +140,17 @@ Service のインスタンスに割り当てる CPU ユニット数。指定可
 
 <div class="separator"></div>
 
-<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>  
-`docker build --platform` で渡すオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>      
+`network` セクションには、Environment の VPC 内の AWS リソースにサービスを接続するためのパラメータが含まれています。Service を VPC に接続することで、[サービス検出](../developing/service-discovery.ja.md)を使用して Environment 内の他の Service と通信したり、[`storage init`](../commands/storage-init.ja.md)で Amazon Aurora などの VPC 内のデータベースに接続することができます。
```

Comment:
In the phrase 'サービスを接続する' the term 'サービス' is used instead of 'Service'. Do you imply the service is not a Copilot service but an App Runner service?

Reply:
That is the meaning of ’Service'. It is my typo error.Should the 'App Runner サービス' in l152 also be 'App Runner Service'?

Reply:
I think it is ok to leave the 'App Runner サービス' on l152 alone because the English sentence is 'the App Runner service routes egress traffic through the private subnets of the VPC.'. The network section contains parameters for Copilot service so the term 'service' would be 'Service' as it is one of Copilot concepts. L152 seems to explain the traffic flows of App Runner VPC networking mode, and [the documentation of AWS App Runner](https://docs.aws.amazon.com/ja_jp/apprunner/latest/dg/architecture.html#architecture.concepts) translates the term 'App Runner service' as 'App Runner サービス'.